### PR TITLE
docs: add English version of conf

### LINKED ISSUE #1493: doc translation (Chinese to English): conf
- Translate the Chinese version to English: https://openmldb.ai/docs/zh/v0.4/deploy/conf.html  (you can download as a markdown file from the button at the top-right corner)
- Save the English version to docs/en/conf.md

### PR DESCRIPTION
**Pull request by cswjd**
docs: add English version of conf
close #1493

### CODE REVIEW

**Comment by cswjd** [comment]
fixes #1493

File: docs/en/conf.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+# Configuration file description
```
Comment (lumianph) [inline docs/en/conf.md]: Configuration Files

File: docs/en/conf.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+## nameserver configuration file conf/nameserver.flags
```
Comment (lumianph) [inline docs/en/conf.md]: ## The Configuration File for Nameserver: conf/nameserver.flags

File: docs/en/conf.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+```
+# nameserver.conf
+--endpoint=127.0.0.1:6527
+--role=nameserver
+# If you are deploying the standalone version, you do not need to configure zk_cluster and zk_root_path, just comment these two configurations. Deploying the cluster version needs to configure these two items, and the two configurations of all nodes in a cluster must be consistent
+#--zk_cluster=127.0.0.1:7181
+#--zk_root_path=/openmldb_cluster
+# The address of the tablet needs to be specified in the stand-alone version, and this configuration can be ignored in the cluster version
```
Comment (lumianph) [inline docs/en/conf.md]: stand-alone => standalone

File: docs/en/conf.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+--tablet=127.0.0.1:9921
+# Configure log directory
+--openmldb_log_dir=./logs
+# Configure whether to enable automatic recovery. If it is enabled, the node will automatically perform the leader switch if it hangs, and the data will be automatically restored after the node process starts.
+--auto_failover=true
+
+# Configure the thread pool size, no need to modify
+#--thread_pool_size=16
+# Configure the number of failed retries, the default is 3
```
Comment (lumianph) [inline docs/en/conf.md]: the number of failed retries 
=>
the number of retry attempts

File: docs/en/conf.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+#--request_max_retry=3
+# Configure the request timeout, the default is 12 seconds
+#--request_timeout_ms=12000
+# Configure the retry interval when the request is unreachable, generally do not need to be modified
+#--request_sleep_time=1000
+# Configure the zookeeper session timeout in milliseconds
+--zk_session_timeout=10000
+# Configure the zookeeper health check interval, the unit is milliseconds, generally do not need to be modified
+#--zk_keep_alive_check_interval=15000
+# Configure the timeout period for tablet heartbeat detection, the default is 1 minute. If the tablet is still connected after this time, the nameserver considers that the tablet is unavailable and will perform the operation of offline the node
```
Comment (lumianph) [inline docs/en/conf.md]: is still connected => is still unreachable
(we have the typo in Chinese doc)

File: docs/en/conf.md [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+#--name_server_task_pool_size=8
+# The number of concurrent execution of high-availability tasks
+#--name_server_task_concurrency=2
+# The maximum number of concurrent execution of high-availability tasks
+#--name_server_task_max_concurrency=8
+# Check the waiting time of the task when executing the task
+#--name_server_task_wait_time=1000
+# The maximum time to execute the task, if it exceeds, it will log
+#--name_server_op_execute_timeout=7200000
+# Get the time interval of the task
```
Comment (lumianph) [inline docs/en/conf.md]: The time interval of receiving the status of the next task
(the Chinese doc is also a bit confusing)

File: docs/en/conf.md [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+#--get_task_status_interval=2000
+# Get the time interval for table status
```
Comment (lumianph) [inline docs/en/conf.md]: The time interval of receiving the status of the next table

**Review by lumianph — CHANGES_REQUESTED** [review]
this is my first batch of comments

File: docs/en/conf.md (hunk 7 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 6 above)
Comment (cswjd) [reply docs/en/conf.md]: ok. Thank you.

File: docs/en/conf.md (hunk 5 above)
Comment (cswjd) [reply docs/en/conf.md]: ok. Thank you.

File: docs/en/conf.md (hunk 4 above)
Comment (cswjd) [reply docs/en/conf.md]: ok. Thank you.

File: docs/en/conf.md (hunk 3 above)
Comment (cswjd) [reply docs/en/conf.md]: ok. Thank you.

File: docs/en/conf.md (hunk 2 above)
Comment (cswjd) [reply docs/en/conf.md]: ok. Thank you.

File: docs/en/conf.md (hunk 1 above)
Comment (cswjd) [reply docs/en/conf.md]: ok. Thank you.

File: docs/en/conf.md [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+
+# Create the default number of replicas for the table
+#--replica_num=3
+# Create the default number of shards for a table
+#--partition_num=8
+# The default number of replicas for system tables
+--system_table_replica_num=2
+```
+
+## tablet configuration file conf/tablet.flags
```
Comment (lumianph) [inline docs/en/conf.md]: ## The Configuration File for Tablets: conf/tablet.flags

File: docs/en/conf.md [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+# Startup ip/domain name and port number
+--endpoint=127.0.0.1:9921
+# The role to start, cannot be modified
+--role=tablet
+
+# If you start the cluster version, you need to specify the address of zk and the node path of the cluster in zk
+#--zk_cluster=127.0.0.1:7181
+#--zk_root_path=/openmldb_cluster
+
+# Configure the thread pool size, it is recommended to be consistent with the number of cpu cores
```
Comment (lumianph) [inline docs/en/conf.md]: cpu => CPU

File: docs/en/conf.md [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+--openmldb_log_dir=./logs
+
+# binlog conf
+# Binlog wait time when no new data is added
+#--binlog_coffee_time=1000
+# Master-slave matching offset waiting time
+#--binlog_match_logoffset_interval=1000
+# Whether to notify the follower to synchronize immediately when data is written
+--binlog_notify_on_put=true
+# The maximum size of the binlog file, in M
```
Comment (lumianph) [inline docs/en/conf.md]: M => MB

File: docs/en/conf.md [hunk 11]
```diff
@@ -0,0 +1,189 @@
...
+--binlog_single_file_max_size=2048
+# Master-slave synchronization batch size
+#--binlog_sync_batch_size=32
+# The interval between binlog sync and disk, in milliseconds
+--binlog_sync_to_disk_interval=5000
+# If there is no new data synchronization wait time, in milliseconds
```
Comment (lumianph) [inline docs/en/conf.md]: The wait time when there is no new data synchronization, in milliseconds

File: docs/en/conf.md [hunk 12]
```diff
@@ -0,0 +1,189 @@
...
+# binlog filename length
+#--binlog_name_length=8
+# The interval for deleting binlog files, in milliseconds
+#--binlog_delete_interval=60000
+# Whether binlog enables crc verification
+#--binlog_enable_crc=false
+
+# Thread pool size for performing io-related operations
+#--io_pool_size=2
+# Execute the thread pool size for tasks such as deleting tables, sending snapshots, load snapshots, etc.
```
Comment (lumianph) [inline docs/en/conf.md]: The thread pool size for tasks such as ...

File: docs/en/conf.md [hunk 13]
```diff
@@ -0,0 +1,189 @@
...
+#--task_pool_size=8
+# Configure the data directory, multiple disks are separated by English symbols
```
Comment (lumianph) [inline docs/en/conf.md]: separated by commas

File: docs/en/conf.md [hunk 14]
```diff
@@ -0,0 +1,189 @@
...
+# snapshot conf
+# Configure the time to do snapshots, the time of day. For example, 23 means taking a snapshot at 23 o'clock every day.
+--make_snapshot_time=23
+# Check interval for snapshots
+#--make_snapshot_check_interval=600000
+# Set the offset threshold of the snapshot, if the offset difference from the last snapshot is less than this value, no new snapshot will be generated
+#--make_snapshot_threshold_offset=100000
+# snapshot thread pool size
+#--snapshot_pool_size=1
+# Whether snapshot compression is enabled. Can be set to off, zlib, snappy
```
Comment (lumianph) [inline docs/en/conf.md]: Whether snapshot compression is enabled, which can be set to off, zlib, snappy

File: docs/en/conf.md [hunk 15]
```diff
@@ -0,0 +1,189 @@
...
+#--snapshot_compression=off
+
+# garbage collection conf
+# The time interval for performing expired deletion, in minutes
+--gc_interval=60
+# Thread pool size to perform expired deletion
+--gc_pool_size=2
+
+# send file conf
+# Maximum number of retries to send a file
```
Comment (lumianph) [inline docs/en/conf.md]: The maximum number of retry attempts to send a file

File: docs/en/conf.md [hunk 16]
```diff
@@ -0,0 +1,189 @@
...
+#--send_file_max_try=3
+# block size when sending files
+#--stream_block_size=1048576
+# Bandwidth limit when sending files, the default is 20M/s
+--stream_bandwidth_limit=20971520
+# Maximum number of retries for rpc requests
```
Comment (lumianph) [inline docs/en/conf.md]: The maximum number of retry attempts for rpc requests

File: docs/en/conf.md [hunk 17]
```diff
@@ -0,0 +1,189 @@
...
+#--request_max_retry=3
+# rpc timeout, in milliseconds
+#--request_timeout_ms=5000
+# If an exception occurs, the retry wait time, in milliseconds
+#--request_sleep_time=1000
+# Retry wait time for file sending failure, in milliseconds
+#--retry_send_file_wait_time_ms=3000
+#
+# table conf
+# The maximum height of the first level jump table
```
Comment (lumianph) [inline docs/en/conf.md]: jump table => skip list

File: docs/en/conf.md [hunk 18]
```diff
@@ -0,0 +1,189 @@
...
+#--skiplist_max_height=12
+# The maximum height of the second level jump table
```
Comment (lumianph) [inline docs/en/conf.md]: jump table => skip list

File: docs/en/conf.md [hunk 19]
```diff
@@ -0,0 +1,189 @@
...
+# loadtable
+# The number of data bars to submit a task to the thread pool when loading
+#--load_table_batch=30
+# Number of threads to load snapshot files
+#--load_table_thread_num=3
+# The maximum queue length of the load thread pool
+#--load_table_queue_size=1000
+```
+
+## apiserver configuration file conf/tablet.flags
```
Comment (lumianph) [inline docs/en/conf.md]: The Configuration File for APIServer: conf/tablet.flags

**Review by lumianph — CHANGES_REQUESTED** [review]
thank you for your contribution!

File: docs/en/conf.md (hunk 19 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you!

File: docs/en/conf.md (hunk 18 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 17 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 16 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 15 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 12 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 11 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 10 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 9 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md (hunk 8 above)
Comment (cswjd) [reply docs/en/conf.md]: Thank you.

File: docs/en/conf.md [hunk 20]
```diff
@@ -94,13 +94,13 @@
...
 --binlog_notify_on_put=true
-# The maximum size of the binlog file, in M
+# The maximum size of the binlog file, in MB
 --binlog_single_file_max_size=2048
 # Master-slave synchronization batch size
 #--binlog_sync_batch_size=32
 # The interval between binlog sync and disk, in milliseconds
 --binlog_sync_to_disk_interval=5000
-# If there is no new data synchronization wait time, in milliseconds
+# Th wait time when there is no new data synchronization, in milliseconds
```
Comment (lumianph) [inline docs/en/conf.md]: Th => The

File: docs/en/conf.md (hunk 13 above)
Comment (lumianph) [reply docs/en/conf.md]: separated by English symbols => separated by commas

**Review by lumianph — CHANGES_REQUESTED** [review]
pls help address the three more minor issues

File: docs/en/conf.md (hunk 20 above)
Comment (cswjd) [reply docs/en/conf.md]: Ok, thank you.

**Review by lumianph — APPROVED** [review]
LGTM
Thanks!

**Pull request merged**